index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+      <figure id="img-div">
+        <img
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt=""
+          id="image"
+        />
+        <figcaption id="img-caption">
+          "Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on
+          hunger."
+        </figcaption>
+      </figure>
+      <div id="tribute-info">
+        <h2>Here's a time line of Dr. Borlaug's life:</h2>
+        <ul>
+          <li>
+            1914 - Born in Cresco, Iowa
+          </li>
+          <li>
+            1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"
+          </li>
+          <li>
+            1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."
+          </li>
+          <li>
+            1937 - Finishes university and takes a job in the US Forestry Service
+          </li>
+          <li>
+            1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.
+          </li>
+          <li>
+            1941 - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.
+          </li>
+          <li>
+            1941 - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.
```

Comment:
### Your Review
![missing-why](https://user-images.githubusercontent.com/93039359/177151634-632caf5e-71f0-4c26-93f0-ae5cb10e2564.png)

- [ ] Great that you have requested a change here. :+1: But you didn't add any reason why the student should use a `section` tag.
- [ ] Also, it is not very clear where you want the student to put the `section` tag. Please clarify that :+1:
- [ ] It sounds like you are ordering the student to apply this change. Please use something like It would be better if you use a section tag instead of a `div` tag with class 'tribute-info' :+1: